### PR TITLE
Milestone: Create Home, About and Quotes Page

### PR DESCRIPTION
**Pull request by Kashif-Saleem-Ghuman**
In this milestone: 
I have used react routing to create a single-page application for the three pages home, about, and quotes.

### CODE REVIEW

**Comment by omarsalem7** [comment]
Hi @Kashif-Saleem-Ghuman 😊, 

While you made a great effort in this project, I cannot proceed to review your code.

## Invalid Code Review Request 🛑 💀 

You have submitted a project, that missing key features, and future features sections in your readme, kindly add them and add 1-3 potential sentences for each section.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by Kashif-Saleem-Ghuman** [comment]
@omarsalem7  Thank you for the feedback. I have made changes and re submitting now.

File: README.md [hunk 1]
```diff
@@ -35,11 +39,55 @@ Make additions, subractions, multiplications, divisions using this app.
...
+
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+#### 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **Calculate sin,cos and tan**
+- [ ] **Calculate square root and to the power**
```
Comment (gemmen29) [inline README.md:88]: - [ ] Kindly add all future features under one section instead of splitting them into two sections in your README.
image
image

**Review by gemmen29 — APPROVED** [review]
Hi @Kashif-Saleem-Ghuman, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 



## Optional suggestions:
- [ ] _Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Kashif-Saleem-Ghuman** [comment]
@gemmen29 Thank you for approving my project and providing valuable feedback. I have made the correction.

**Pull request merged**